package.json

```diff
@@ -12,11 +12,27 @@
     "node": ">=14.17.5",
     "npm": ">=7.16.0"
   },
-  "workspaces": [
+  "workspacesPatterns": [
     "packages/*",
     "configs/*",
     "scripts"
   ],
+  "workspaces": [
```

Comment:
And then this script will manage the workspaces prop based on workspacesPatterns.